src/openklant/components/klantinteracties/tests/test_admin.py

```diff
@@ -90,3 +94,35 @@ def test_search(self):
         self.assertContains(search_response, digitaal_adres_persoon.get_full_name())
         self.assertNotContains(search_response, nummer_persoon.get_full_name())
         self.assertNotContains(search_response, uuid_persoon.get_full_name())
+
+    @disable_admin_mfa()
+    def test_digitaal_adres_inline(self):
+        """
+        Regression test for #226
+
+        betrokkene should be optional
+        """
+
+        SuperUserFactory(username="admin", password="admin")
+        self._login_user(username="admin", password="admin")
+
+        partij = PartijFactory(soort_partij="persoon", voorkeurs_digitaal_adres=None)
+        PersoonFactory(partij=partij)
+        url = reverse("admin:klantinteracties_partij_change", args=[partij.pk])
+
+        response = self.app.get(url)
+
+        form = response.form
+        form["digitaaladres_set-TOTAL_FORMS"] = 1
+        add_dynamic_field(form, "digitaaladres_set-0-omschrijving", "description")
+        add_dynamic_field(form, "digitaaladres_set-0-soort_digitaal_adres", "email")
+        add_dynamic_field(form, "digitaaladres_set-0-adres", "email@example.com")
```

Comment:
As an alternative to having to add these fields, you could maybe create a DigitaalAdres with a factory to ensure the form is there, and then try to save it again without a betrokkene, but either way is fine to me